src/app.ts

```diff
@@ -13,14 +14,56 @@ interface TemperatureSummary {
   average: number
 }
 
+interface TemperaturePerDay {
+  [cityID: string]: {
+    summary: TemperatureSummary
+    count: number
+    total: number
+  }
+}
+
+const temps: TemperaturePerDay = {}
+
 export function processReadings(readings: TemperatureReading[]): void {
   // add here your code
+  readings.forEach((reading: TemperatureReading) => {
```

Comment:
* If `readings` is already known to be an array of `TemperatureReading` then it is redundant said reading is a `TemperatureReading` 
* For more variables readability replace `reading` for `read`